.github/workflows/rollbackDB.yml

```diff
@@ -3,24 +3,35 @@ name: Rollback DB
 on:
   workflow_dispatch:
     inputs:
-      tag:
-        description: 'The Liquibase tag to roll back to'
-        required: true
       environment:
-        description: 'The environment of the DB that should be rolled back. Options: (demo dev dev2 dev3 dev4 dev5 dev6 dev7 pentest test training stg prod)'
+        description: |
+          The environment of the database.
         required: true
-
-concurrency:
-  group: db-rollback
+        type: choice
+        options: [demo, dev, dev2, dev3, dev4, dev5, dev6, dev7, pentest, test, training, stg, prod]
+        default: dev
+      action:
+        description: |
+          The liquibase action to run.
+        required: true
+        type: choice
+        options: [validate, clear-checksums, rollback]
+        default: validate
+      tag:
+        description: |
```

Comment:
I'm wondering if there's a way to conditionally require this field for when a user selects the rollback option? 🤔

Reply:
i ran it without a tag to see what would happen https://github.com/CDCgov/prime-simplereport/actions/runs/4893957156/jobs/8737569806

Reply:
Good thinking, I think the best we could do would be to check in bash before we deploy the terraform. I think I'll leave that for a future iteration for now.